cypress/e2e/fee-fine/overdue-fine-returned-triggers.cy.js

```diff
@@ -348,7 +348,7 @@ describe('Overdue fine', () => {
 
       // need to wait so item can get overdue status
       // eslint-disable-next-line cypress/no-unnecessary-waiting
-      cy.wait(100000);
+      cy.wait(120000);
       cy.visit(TopMenu.checkInPath);
       CheckInActions.checkInItem(itemData.barcode);
       CheckInActions.verifyLastCheckInItem(itemData.barcode);
```

Comment:
This test has issue like 'Expected to return a value at the end of arrow function'. Please fix it.

Reply:
done